graph/quadstore.go

```diff
@@ -44,7 +44,7 @@ type Value interface{}
 type QuadStore interface {
 	// The only way in is through building a transaction, which
 	// is done by a replication strategy.
-	ApplyDeltas([]Delta) error
+	ApplyDeltas([]Delta, bool, bool) error
```

Comment:
I'll take it. I might have done a struct for this case, but (at the moment) it's two bools, and may get updated. So, this may get upgraded to full options or to a struct if we add another flag, but for now, I'm willing to go with it.